Makefile

```diff
@@ -2,17 +2,27 @@ BIN = ./node_modules/.bin
 
 SRC = lib
 TESTS = test
+DIST = dist
 
 .PHONY: test watch clean
 
 test: clean
+	make build
 	$(BIN)/jshint --reporter node_modules/jshint-stylish/stylish.js $(SRC) $(TESTS)
 	$(BIN)/jscs $(SRC) $(TESTS)
 	$(BIN)/karma start --single-run
 	make clean
 
-watch:
+watch: build
 	$(BIN)/karma start
```

Comment:
hm, will this rebuild on file changes? What about something like https://www.npmjs.com/package/karma-rollup-preprocessor?